docs/quickstart.md

```diff
@@ -0,0 +1,173 @@
+# Quickstart
+
+Welcome! This quickstart guide will get you up and running with `lakefs-spec` by showing you how to
+
+1. [Install the `lakefs-spec` package](#installing),
+1. [spin up a local lakeFS server](#spinning-up-a-local-lakefs-instance),
+1. [create a lakeFS repository for experimentation](#create-a-lakefs-repository), and
+1. [perform basic file system operations](#using-the-lakefs-fsspec-file-system)
+in a lakeFS repository using `lakefs-spec`.
+
+??? info "Prerequisites"
```

Comment:
Just for clarification, because I can see the syntax being confusing: the `???` is a special markup for callouts in mkdocs, the block is rendered in the generated documentation as such:

![image](https://github.com/appliedAI-Initiative/lakefs-spec/assets/322624/d4d44166-3ed5-4f72-9608-1f1c4328ab0d)

Reply:
Addressed in e999e795ed4fca501a99d949fbdb6f491262133f